controller.go

```diff
@@ -192,12 +195,12 @@ func New(cfgOptions ...config.Option) (NetworkController, error) {
 			dcfg = c.makeDriverConfig(i.ntype)
 		}
 
-		if err := drvRegistry.AddDriver(i.ntype, i.fn, dcfg); err != nil {
+		if err := drvRegistry.AddDriver(i.ntype, i.fn, dcfg, c.plugingetter); err != nil {
```

Comment:
Done. I've also split the PR into godep vendoring and plugin related code, so its easier to review.